enroll/service.go

```diff
@@ -8,6 +8,9 @@ import (
 
 type Service interface {
```

Comment:
Each one of these methods should have a comment in the form of 
```
// MethodName ...
```
That explains what the method does. It would be especially useful with the OTA methods since they're a bit cryptic to begin with.